style.css

```diff
@@ -420,3 +420,103 @@ ul {
   color: #e2505c;
   border: 1px solid #e2505c;
 }
+
+.form-section {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  grid-gap: 24px;
+  background-image: url(./images/image1.png);
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Please make sure that flexbox is used to lay out the elements in the form section :pray: . But as you can see, you used grid.